.github/workflows/ci.yml

```
@@ -150,9 +156,10 @@ jobs:
     with:
       clang-version: 20
       cmake-args: -DBUILD_MQT_DDSIM_BINDINGS=ON
-      files-changed-only: true
+      files-changed-only: false
```

Set this back to `true` before merging. 

```suggestion
      files-changed-only: true
```